apps/meteor/server/models/raw/LivechatContacts.ts

```diff
@@ -56,12 +56,24 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 					'channels.visitor.source.type': 1,
 					'channels.visitor.source.id': 1,
 				},
+				name: 'visitorAssociation',
 				unique: false,
 			},
 			{
 				key: {
-					channels: 1,
+					'channels.field': 1,
+					'channels.value': 1,
+					'channels.verified': 1,
 				},
+				partialFilterExpression: { 'channels.verified': true },
+				name: 'verificationKey',
+				unique: false,
+			},
+			{
+				key: {
+					preRegistration: 1,
+				},
+				name: 'preRegistration',
```

Comment:
I guess we can make this index `sparse`, since there will be less records with it set.. right?
```suggestion
				sparse: true,
```